### ISSUE TITLE
[A11y] Explore Collections - Grid - Headings

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1883]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

- The `<h1>` should match the page title.  Here, we have two level one headings (one is hidden) and neither one matches the page title.
- There are two hidden level-two headings.  I believe these are okay if the first level one heading is fixed.  It would be good to change these so they aren't all-caps.
- The second level one heading, "Search Results", should be a level 2 heading.
- **Update**: I don't know how I missed this, but the collections themselves are all level 3 headings.  These aren't marking any kind of section, and should not be headings.

### Suggested Fix

- Generally speaking, the H1 should match the page title if possible
- Make "search results" a level two heading
- Remove the use of `<h3>` (or any heading tag) from the results.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Somebody has to inspect the DOM or use a tool to view the heading outline

**Comment by jechols** [comment]
Almost, but the collections are still level 3 headings but have no content, so they shouldn't be headings
